solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -290,6 +297,14 @@ contract RandomBeacon is Ownable {
         relay.setRelayEntrySubmissionFailureSlashingAmount(1000e18);
     }
 
+    /// @notice Updates the minimum authorization amount.
+    function updateMinimumAuthorization(uint96 minAuthorization)
+        external
+        onlyOwner
+    {
+        minimumAuthorization = minAuthorization;
+    }
```

Comment:
We should emit an event and assert it is emitted in `updateMinimumAuthorization` context in `RandomBeacon.Parameters.test.ts`.

Reply:
I'm a bit confused actually, who can call this function? I thought it's governance through the RandomBeaconGovernance contract. If yes, won't we duplicate the event which is in `beginMinimumAuthorizationUpdate` and later in `MinimumAuthorizationUpdated`?

Reply:
We may change the owner one day and we still want to be able to locate when the given parameter changed. That's why emit another event in `RandomBeacon`.

Reply:
fd5ff63